README.md

```diff
@@ -96,3 +96,65 @@ Useful sub-commands include:
 
 To automatically reformat all code to comply with
 some (but not all) of the style checks, run `tox -e format`.
+
+
+## Importing / Exporting Data
```

Comment:
This looks like documentation for how to use it. It is great! But I definitely agree with you that we should break up this huge main readme. We should separate it out into [development docs](https://github.com/OpenImaging/miqa/blob/master/development.md), [deployment docs](https://github.com/OpenImaging/miqa/blob/master/prod/README.md) and [usage docs](https://github.com/OpenImaging/miqa/blob/master/how-to-import-json.md). These files might need to be renamed or moved. https://github.com/OpenImaging/miqa/blob/master/prod/DEVELOPMENT.md might need to be merged with either development or deployment one? They should all be linked from the main readme, which should be quite short now, and be more an advertisement. I don't think we need to link to [NN docs](https://github.com/OpenImaging/miqa/blob/master/miqa/learning/README.md).

Reply:
Reorganized in 7f5e233